File: Go/002-Even_Fibonacci_numbers.go
package main

import "fmt"

func fibo(n int) int {
	if n == 0 {
		return 0
	} else if n == 1 {
		return 1
	} else {
		return fibo(n - 1) + fibo(n - 2)
	}
}

func main() {
	sum := 0
	for i:=0; fibo(i) < 4000000; i++{
		if fibo(i) % 2 == 0 {
			sum += fibo(i)
		}
	}
	fmt.Printf("%d", sum)
}

File: Python/002_Even_Fibonacci_numbers.py
def fibo(n1, n2, res):
    if n1 > 4000000:
        return res
    return fibo(n2, n2 + n1, res + n2 if n2 % 2 == 0 else res)

print( fibo(1, 1, 0) )
print(4613732)
